### DISCUSSION TITLE
Fill type for haul trucks from excavators/loaders/etc

### DISCUSSION THREAD

**Discussion by venominc831** [VertexDezign/TargetFillLevel#2] [Ideas]
Would it be possible to modify this so that a mod excavator, wheel loader, shovel loader etc could be able to see the fill level of haul trucks like a cat 395b filling up a cat 737?

**Comment by schliesser** [comment]
Heyho @venominc831 

Great idea!

I'm working on a feature like that for wheel loaders when filling the biogas plant etc. You can test this with the latest development version of this mod. Download here: https://github.com/VertexDezign/TargetFillLevel/releases/tag/latest

Can you test it and give me some feedback?

Thank you very much.

**Comment by venominc831** [comment]
Sure, I'll figure out a way to test it one way or another. I'll see if we can get it in our MP .

**Reply by schliesser** [reply]
Heyho @venominc831 

had you time to test it and how are the results?

**Reply by venominc831** [reply]
To be honest, I've been super busy over the past two weeks, but I'm going to load it up now to try to test it.

**Reply by venominc831** [reply]
just finished doing some testing, seems like it works with some of the base game stuff, but couldn't see it with any of the modded loaders or excavators I have

**Reply by schliesser** [reply]
Can you share those mods with me somehow?

**Reply by venominc831** [reply]
Sorry, I've been really busy with real life. I was able to test this on a different map with different equipment and it's been showing up for different excavators and trucks. If I run across some it doesn't work on, I'll let you know.

**Reply by schliesser** [reply]
Awesome to hear, that it works.
Hopefully, the new version will be released the next days on modhub.

**Comment by schliesser** [comment]
Heyho @venominc831 
just wanted to let you know the new version (+ hotfix) has been released on ModHub today as version 2.0.1.1. https://www.farming-simulator.com/mod.php?mod_id=231717

I'll close the discussion as the feature is now implemented. Feel free to reopen or open a new one if you think it's not yet fulfilled.

Have a nice farming day 🚜
